code/game/gamemodes/changeling/powers/shuriken.dm

```diff
@@ -0,0 +1,73 @@
+/datum/action/changeling/shuriken
+	name = "Bone Shuriken"
+	desc = "We evolve the ability to break off shards of our bone, and shape them into 2 shurikens. Costs 15 chemicals per shuriken."
+	helptext = "This will damage our arms, and be noticable to nearbye people. Shurikens will be dulled after hitting someone. 30 chemicals are required to use this ability, but you'll only be charged 15 per shuriken you make, and refunded 15 for those you do not."
```

Comment:
Instead of fiddling with refunds, let's have it cost 15 and make a single shuriken at a time?